index.html

```diff
@@ -7,41 +6,43 @@
     <title>NoteKeeper</title>
     <link rel="icon" href="data:," />
     <link rel="stylesheet" href="sanitize.css">
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="global.css">
+    <link rel="stylesheet" href="styles.css">
     <script src="script.js" defer></script>
 </head>
 <body>
     <header>
         <h1 class="mainTitle">NoteKeeper</h1>
     </header>
-    <main>
+    <main class="mainContent">
         <section aria-labelledby="newNoteFormHeading">
-            <h2 class="newNoteFormHeading">New Note Form</h2>
-            <form class="newNoteForm" aria-describedby="newNoteInstructions">
+            <h2 class="formAndListTitle">New Note Form</h2>
+            <form class="noteBase newNoteForm" aria-describedby="newNoteInstructions">
                 
                 <label for="noteTitle">Title</label>
```

Comment:
Attibute `for` of labels is not correctly bound to input identifiers, so a11y is not ensured.

On the other hand, the `aria-required="true"` attribute is used to indicate that an element, usually a form control, is required. This attribute is primarily used to communicate to assistive technologies (like screen readers) that an input field must be filled out. 

However, if you already have the standard HTML `required` attribute on an input field, the `aria-required="true"` attribute is not necessary. The `required` attribute alone is sufficient, as modern screen readers typically recognize the `required` attribute and will announce that the field is mandatory.

In your implementation:

```html
<input type="text" id="noteTitle" class="noteTitle" name="noteTitle" required aria-required="true">
```

- **`required`**: This is the native HTML attribute that will both visually indicate the field is required (with some default styles) and alert screen readers that this field needs to be filled out.
- **`aria-required="true"`**: This is redundant when used alongside the `required` attribute. It doesn’t add any additional functionality for screen readers beyond what `required` already provides.

You can safely remove `aria-required="true"` in this case. Just using `required` will cover accessibility needs. This keeps the code clean and avoids unnecessary redundancy.

Also remember than input must be properly auto-closed.

```suggestion
                <label for="noteTitle">Title</label>
                <input type="text" id"noteTitle" class="noteTitle" name="noteTitle" required />
```

- [ ] ⚠️ fix comments